components/authz-service/server/v2/authz.go

```diff
@@ -68,6 +68,22 @@ func (s *authzServer) ProjectsAuthorized(
 		projects = req.ProjectsFilter
 	}
 
+	// this call checks for all allowed projects (without considering deny)
+	if version.Minor == api.Version_V1 && len(req.ProjectsFilter) == 0 {
```

Comment:
I wonder if we're due for a helper that's something like `if beta2p1` or something . . . I know we *shouldn't* fall in here without `Major == v2` but still . . . this logic asks that question but makes the reader really have to 🤔 about it